cmd/gh/main.go

```diff
@@ -176,6 +176,12 @@ func mainRun() exitCode {
 
 		printError(stderr, err, cmd, hasDebug)
 
+		if strings.Contains(err.Error(), "Incorrect function") {
+			fmt.Fprintln(stderr, "You appear to be running in MinTTY without pseudo terminal support.")
+			fmt.Fprintln(stderr, "To learn about workarounds for this error, gh help mintty")
```

Comment:
Could we say: `To learn …, run:  gh help mintty` to separate the command from the prose

Reply:
ah, typo